app/src/RoomClient.js

```diff
@@ -2254,7 +2256,7 @@ export default class RoomClient
 		{
 			const levelArea = width * height / (resolutionScalings[i] ** 2);
 
-			if (availableArea >= levelArea)
+			if (availableArea >= levelArea * adaptiveScalingFactor * adaptiveScalingFactor)
```

Comment:
Why 2 times * adaptiveScalingFactor ? This is already the are, so we need to multiply only 1 time to get 0.75 of the are, right?

Reply:
If you multiply for adaptiveScalingFactor only one time, you are scaling the area required to trigger a level update; using adaptiveScalingFactor ** 2 you are scaling the width and the height, so it may be more intuitive for administrators.

Reply:
OK, but then in the example you should write both width and height, and that each of them times the factor multiplied is.

Reply:
With the latest commit I've changed the detection algorithm checking if `viewportWidth >= levelWidth || viewportHeight >= levelHeight` instead of using area.

Reply:
Perfect